webapp/channels/src/components/invitation_modal/invitation_modal.scss

```diff
@@ -12,5 +12,13 @@
         justify-content: center;
         margin: auto;
         transform: translate(-50%, -50%) !important;
+
+    }
+    &__header {
+        padding-left: 0 !important;
```

Comment:
Are these because the styling of this modal doesn't match GenericModal currently? Perhaps we could talk to UX about changing this one to match the designs elsewhere since we should be trying to standardize these more

Reply:
That makes sense. Thanks for the explanation!